code/src/Components/ThoughtForm.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const ThoughtForm = ({ onFormSubmit, newThought, setNewThought }) => {
+  return (
+    <form className="form card" onSubmit={onFormSubmit}>
+      <label htmlFor="newThought">What's making you happy right now?</label>
+      <textarea
+        className={newThought.length > 140 ? "red-text" : ""}
```

Comment:
This part with a new way of using className I asked about during your demo and saw in the material for this week that we are going to use it, great job with using it in this project!